src/components/Menu.vue

```diff
@@ -366,6 +379,12 @@ export default {
     margin-left: 10px;
   }
 
+  span.nomlogSummary {
+    color: $townsfolk;
+    margin-top: 7px;
+    margin-left: 10px;
```

Comment:
Aren't those already inherited from [line 378](https://github.com/bra1n/townsquare/pull/125/files#diff-e065371e6591864d797ae6468724bc4bcb3d329b950921fdf19f4c18e07dc500R378-R379)?

Reply:
Seems like they are. I was originally looking at different separations/placements and didn't remove them. I will remove them.